src/assets/sass/variables/colors.scss

```
@@ -69,3 +69,7 @@ $orange: #d97706;
 
 $olive-dark: #353428;
 $white: #fff;
+
+$dropdown-text: #1d192b;
```

Stylelint (17.5.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)